content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -404,6 +406,71 @@ instances:
 ### Serverless environment
 
 For more information about AWS Lambda functions, see [how to connect your Lambda telemetry using tags][15].
+
+### OpenTelemetry
+
+When using OpenTelemetry, map the following [resource attributes][16] to their corresponding Datadog conventions:
+
+| OpenTelemetry convention | Datadog convention |
+| --- | --- |
+| `deployment.environment` | `env` |
+| `service.name` | `service` |
+| `service.version` | `version` |
+
+<div class="alert alert-warning">Do not use Datadog-specific tags like <code>DD_SERVICE</code>, <code>DD_ENV</code> or <code>DD_VERSION</code> in your OpenTelemetry configuration.</div>
+
+{{< tabs >}}
+{{% tab "Environment variables" %}}
+
+To set resource attributes using environment variables, set `OTEL_RESOURCE_ATTRIBUTES` with the appropriate values:
```

Comment:
I'm not sure if this is the correct env variable, or if they should just use OTEL_SERVICE_NAME?

Reply:
Both would work for setting `service.name`, `OTEL_RESOURCE_ATTRIBUTES` is more general